README.md

```diff
@@ -7,14 +7,10 @@
 # To build this application:
 
 ```
-$ mkdir wgo
-$ cd wgo
-$ export GOPATH=`pwd`
-$ export GOBIN=$GOPATH/bin
-$ export PATH=$GOBIN:$PATH
 $ go get github.com/skelterjohn/wgo
 $ go install github.com/skelterjohn/wgo
```

Comment:
That's right - my PR to this repo simplifies the instructions quite a bit.

On Thu, May 21, 2015 at 9:56 AM, Glenn Lewis <notifications@github.com>
wrote:

> In README.md
> <https://github.com/jbuberel/example_wgo/pull/1#discussion_r30803773>:
>
> > @@ -7,14 +7,10 @@
> >  # To build this application:
> >
> >  ```
> > -$ mkdir wgo
> > -$ cd wgo
> > -$ export GOPATH=`pwd`
> > -$ export GOBIN=$GOPATH/bin
> > -$ export PATH=$GOBIN:$PATH
> >  $ go get github.com/skelterjohn/wgo
> >  $ go install github.com/skelterjohn/wgo
>
> Why do we need go get followed by go install?
> According to https://golang.org/cmd/go/, a go get also performs an
> install.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jbuberel/example_wgo/pull/1/files#r30803773>.
>